r/Dockerfile

```diff
@@ -34,6 +40,15 @@ RUN R -e "install.packages('covr')"
 RUN R -e "install.packages('xml2')"
 
 
+RUN apt-get install -y libudunits2-dev
+RUN apt-get install -y libgdal-dev
+
+RUN R -e "install.packages('sf')"
+RUN R -e "devtools::install_github('cole-trapnell-lab/monocle3')"
```

Comment:
why is this repeated 2 times? (see line 33)

Reply:
all of this should go to the package.lock file. you can install things from github without devtools. using `renv::install('cole-trapnell-lab/monocle3')` works. and then when snapshotting it adds github as the source.